website/src/app/pages/data-distributions/charts/bar-chart.vl.ts

```diff
@@ -0,0 +1,209 @@
+import { VisualizationSpec } from 'vega-embed';
+import { VariableData, DistributionData } from '../data-distributions.component';
+
+export function createHorizBarSpec(variable: VariableData, distributionData: DistributionData[] = []): VisualizationSpec {
+  return createBarSpec(variable, distributionData, true);
+}
+
+export function createBarSpec(variable: VariableData, distributionData: DistributionData[] = [], flipAxes = false): VisualizationSpec {
+  return {
+    $schema: 'https://vega.github.io/schema/vega-lite/v5.json',
+    datasets: {
```

Comment:
For all specs, move datasets to the end of the spec (this improves the readability when loading in vega editor)